pkg/util/caller/resolver.go

```diff
@@ -120,7 +119,7 @@ func (cr *CallResolver) Lookup(depth int) (file string, line int, fun string) {
 		if len(matches) == 1 {
 			file = matches[0]
 		} else {
-			file = filepath.Join(matches[1:]...)
+			file = path.Join(matches[1:]...)
```

Comment:
The `path` vs `filepath` distinction can be a bit sublte. Can you add a comment explaining that `path.Join` always uses slash separators (as opposed to `filepath.Join`)?